niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/SignaturePermissionTester.java

```diff
@@ -2207,12 +2278,15 @@ public void onError(int error) {
 
         mPermissionTasks.put(permission.NETWORK_MANAGED_PROVISIONING,
                 new PermissionTest(false, Build.VERSION_CODES.Q, () -> {
-                    // IWifiManager#setWifiEnabled
-                    Parcel result = mTransacts.invokeTransact(Transacts.WIFI_SERVICE,
-                            Transacts.WIFI_DESCRIPTOR,
-                            Transacts.setWifiEnabled, mPackageName, true);
-                    if (result.readBoolean() == false) {
-                        throw new SecurityException("Unable to set WIFI enabled");
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
Nit: This check shouldn't be necessary since the min SDK version of the task is set to Android Q.

Reply:
it's for avoidng android lint check. Sdk28(project sdk) does't support Parcel.readBoolean.

Reply:
Thanks!